### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by ilya-korotya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,51 +78,42 @@ The blockchain verification algorithm is used
...
                     resolver := state.ETHResolver{
                         RPCUrl:   "<rpc url>",
                         Contract:  "contract address",
                     }
                     verifier := auth.NewVerifier(verificationKeyloader, loaders.DefaultSchemaLoader{IpfsURL: "ipfs.io"}, resolver)
    ```
+4. FullVerify:
 
+   ```go
+         authResponse, err := verifier.FullVerify(r.Context(), string(tokenBytes),
```
Comment (vmidyllic) [inline README.md:111]: how to init verifier

File: README.md [hunk 2]
```diff
@@ -48,16 +48,21 @@ Circuit public signals marshallers are defined in the go-circuits library.
...
 
+The blockchain verification algorithm is used
 
-
-
+1. Get GIST from the blockchain (address of id state contract and URL must be provided by the caller of the library):
+   1. Empty state is returned - it means that GIST state hasn’t been updated or updated GIST hasn’t been published. We need to retrun an error.
+   2. A non-empty GIST is returned, equal to the GIST is provided by the user, it means the user is using the latest state.
+   3. The non-empty GIST is returned and it’s not equal to the GIST is provided by a user. Gets the transition time of the GIST. The verification party can make a decision if it can accept this state based on that time frame.
+2. Only latest states for user are valid. Any existing issuer state for claim issuance is valid.
```
Comment (vmidyllic) [inline README.md:65]: bu

File: README.md [hunk 3]
```diff
@@ -48,16 +48,21 @@ Circuit public signals marshallers are defined in the go-circuits library.
...
 2. Only latest states for user are valid. Any existing issuer state for claim issuance is valid.
 
+### Verification of GIST
 
+The blockchain verification algorithm is used
 
-
-
+1. Get GIST from the blockchain (address of id state contract and URL must be provided by the caller of the library):
+   1. Empty state is returned - it means that GIST state hasn’t been updated or updated GIST hasn’t been published. We need to retrun an error.
```
Comment (vmidyllic) [inline README.md:62]: remove 1 at all. remove non-empty from 2 /3

Comment (ilya-korotya) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ilya-korotya) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (ilya-korotya) [reply README.md]: done

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**